examples/scripts/quickstart.py

```diff
@@ -52,7 +49,7 @@ def wait_until_train_job_has_completed(client, app):
             if status == TrainJobStatus.COMPLETED:
                 # Train job completed!
                 return True
-            elif status != TrainJobStatus.RUNNING:
+            elif status == TrainJobStatus.STOPPED:
```

Comment:
Any big reason for this change? previously, it handles the case when the train job errored

Reply:
If the SubTrainJobs are all on STARTED status, then the TrainJobStatus would be STARTED which in this case if it is status != TrainJobStatus.RUNNING wud return False? But it could be the case where the SubTrainJob hasn't started running yet (meaning no error).

Reply:
ok